vmck/api.py

```diff
@@ -63,5 +100,6 @@ def view(request, **kwargs):
 urls = [
     path('', route(GET=home)),
     path('jobs', route(POST=create_job)),
+    path('submission', route(POST=create_submission)),
```

Comment:
There should be a single endpoint, `create_job`. Whatever extra information we need to send, we can add it to a key-value dictionary in the request, let's call it `env`, that gets passed on to the control job (`vmck/vagrant-vmck-acs`).